gradle-baseline-java/src/test/groovy/com/palantir/baseline/BaselineCircleCiJavaIntegrationTests.java

```diff
@@ -93,7 +93,7 @@ public void junitIntegrationTest() throws IOException {
                 .buildAndFail();
         assertThat(result.getOutput()).contains("2 tests completed, 1 failed");
 
-        File report = new File(reportsDir, "junit/test/TEST-com.example.MyClassTests.xml");
+        File report = new File(reportsDir, "junit/test/TEST-MyClassTests.xml");
```

Comment:
are file names meaningful here? Does this mean that you have a chance of conflict now?

Reply:
The names aren't meaningful, but I've fixed it to avoid conflicts